docs/accessibility.rst

```diff
@@ -0,0 +1,182 @@
+    class MyPage(PageObject):
+
+        def url(self):
+            return 'https://www.mysite.com/page'
+
+        def axs_audit_rules(self):
+            return ['badAriaAttributeValue', 'imagesWithoutAltText']
+
+
+    class AccessibilityTest(WebAppTest):
+
+        def test_accessibility_on_page(self):
+            page = MyPage(self.browser)
+            page.visit()
+            report = page.do_axs_audit()
+
+            # There was one page in this session
+            self.assertEqual(1, len(report))
+            result = report[0]
+
+            # I have already corrected any accessibility errors on my page
+            # for the rules I defined in the page object, so I will assert
+            # that none exist.
+            self.assertEqual(0, len(result.errors))
+            self.assertEqual(0, len(result.warnings))
+
+
+Trigger the audit passively
+---------------------------
+
+You might already have some bok-choy tests written for your web application.
+Here is an example of one:
+
+
+.. code-block:: python
+
+    from bok_choy.page_object import PageObject
+    class MyPage(PageObject):
+
+        def url(self):
+            return 'https://www.mysite.com/page'
+
+        def axs_audit_rules(self):
+            return ['badAriaAttributeValue', 'imagesWithoutAltText']
+
+        def click_button(self):
+            """
+            Click on the button element (id="button").
+            On my example page this will trigger an ajax call
+            that updates the #output div with the text "yes!"
+            """
+            self.q(css='div#fixture button').first.click()
+            self.wait_for_ajax()
+
+        @property
+        def output(self):
+            """
+            Return the contents of the "#output" div on the page.
+            """
+            text_list = self.q(css='#output').text
+
+            if len(text_list) < 1:
+                return None
+            else:
+                return text_list[0]
+
+    class MyPageTest(WebAppTest):
+
+        def test_button_click_output(self):
+            page = MyPage(self.browser)
+            page.visit()
+            page.click_button()
+
+            self.assertEqual(page.output, 'yes!')
+
+
+You can reuse your existing bok-choy tests in order to navigate through
+the application while at the same time verifying that it is accessibile.
+
+Before running your bok-choy tests, set the environment variable
+VERIFY_ACCESSIBILITY to true.
+
+::
+
+    export VERIFY_ACCESSIBILITY=True
+
+This will trigger an audit, using the rules (and optionally the scope) set in
+the page object definition, whenever a call to wait_for_page() is made.
+
+In the case of the test_button_click_output testcase in the above example,
+an audit will be done at the end of the visit() and click_button() method calls,
+as each of those will call out to wait_for_page().
+
+If any assessibility errors are found, then the testcase will fail with an
+AccessibilityError.
```

Comment:
Could you give an example here? Just to make it easy-peasy for python novices to understand.